src/django/api/services/search.py

```diff
@@ -0,0 +1,24 @@
+import logging
+from .opensearch import OpenSearchServiceConnection
+from .search_interface import SearchInterface
+from opensearchpy.exceptions import OpenSearchException
+
+logger = logging.getLogger(__name__)
+
+
+class OpenSearchService(SearchInterface):
+    def __init__(self):
+        self.client = OpenSearchServiceConnection().client
```

Comment:
I think you can clearly indicate that is private by using two underscores, since it is used only internally.
E.g. `self.__client = OpenSearchServiceConnection().client`

Reply:
Done, pls check